interpreter/llvm/src/include/llvm/Support/CMakeLists.txt

```diff
@@ -38,6 +38,13 @@ set(version_inc "${CMAKE_CURRENT_BINARY_DIR}/VCSRevision.h")
 
 set(get_svn_script "${LLVM_CMAKE_PATH}/GenerateVersionFromCVS.cmake")
 
+# Ugly hack to prevent rebuilding LLVM whenever the git HEAD timestamp
+# changes. This is properly solved by setting LLVM_APPEND_VC_REV to OFF
+# but this only really works once this review is included in our LLVM
+# version: https://reviews.llvm.org/D35377
```

Comment:
It's multiple commits that refactored this thing, D35377 is only the one that actually makes it fully work as intended (we don't even have the setting variable he's using in our CMake code yet), and backporting bigger pieces of CMake code isn't a lot of fun.

It's also just until the next LLVM upgrade where the other commit fixes this properly and we can drop this hack.